src/manifest.chrome.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Cyber Citizenship Coefficient",
+  "description": "Build an Chrome Plugin Extension",
+  "version": "1.0",
```

Comment:
Use 1.0.0

Reply:
Why are there 2?

Reply:
I am using 1.0.0

Reply:
This manifest (not sure why we need 2) says 1.0

Reply:
I deleted the manifest.chrome.json